scripts/cleanup-dev-resources.sh

```
@@ -0,0 +1,102 @@
+# AutoCleanup 태그가 있는 EC2 인스턴스 찾기
+echo "🔍 정리 대상 EC2 인스턴스 검색 중..."
+INSTANCE_IDS=$(aws ec2 describe-instances \
+    --filters "Name=tag:AutoCleanup,Values=true" "Name=instance-state-name,Values=running,stopped" \
+    --query 'Reservations[].Instances[].InstanceId' \
+    --output text)
+
+if [ -z "$INSTANCE_IDS" ]; then
+    echo "✅ 정리할 개발 환경 EC2 인스턴스가 없습니다."
+else
+    echo "🗑️  다음 인스턴스들을 종료합니다: $INSTANCE_IDS"
+    aws ec2 terminate-instances --instance-ids $INSTANCE_IDS
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)